dkist/dataset/tests/test_crop_dataset.py

```diff
@@ -0,0 +1,185 @@
+import pytest
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord, SpectralCoord, StokesCoord
+from astropy.time import Time
+
+gwcs = pytest.importorskip("gwcs", "0.22.2")
```

Comment:
This was the first version to have the fix:
```suggestion
gwcs = pytest.importorskip("gwcs", "0.22.2a2")
```